### DISCUSSION TITLE
How can I include the file that the link point to?

### DISCUSSION THREAD

**Discussion by zousiyu1995** [Witiko/markdown#336] [Q&A]
For example, below is a link in markdown file,

```markdown
- [p34. find first and last position of element in sorted array](./src/p34_find_first_and_last_position_of_element_in_sorted_array.rs)
```

It points to a local code file. My question is how can I include the code file to latex documentation. 

By default, the latex documentation can only generate a link in the PDF, not include the file that the link point to.

**Comment by Witiko** [comment]
Hello. How would you like the code file to be included? Would you like to produce a code listing for the code file at some place in the document or would you like to store the file as an attachment in the PDF document?

**Reply by zousiyu1995** [reply]
I want produce a code listing for the code file.
